protos/chaincode.proto

```diff
@@ -78,10 +78,16 @@ message ChaincodeSpec {
 // TODO: Define `codePackage`.
 message ChaincodeDeploymentSpec {
 
+    enum ExeEnv {
```

Comment:
How about ExecutionEnvironment ? Docker is a container, system chaincode is in-process, next could be external-process,.... they are all chaincode execution environments.